app/script/conversation/ConversationMapper.js

```diff
@@ -99,16 +99,17 @@ z.conversation.ConversationMapper = class ConversationMapper {
    *
    * @param {Array} conversationsData - Conversation data
    * @param {number} [timestamp=1] - Initial timestamp for conversation
+   * @param {boolean} [isTeam] - Type of account for state mapping
    * @returns {Array<Conversation>} Mapped conversation entities
    */
-  mapConversations(conversationsData, timestamp = 1) {
+  mapConversations(conversationsData, timestamp = 1, isTeam) {
```

Comment:
Let's set a default value for `isTeam`, otherwise we'll have a required value after an optional value.

Reply:
Check the JSDoc. It is optional. It will work without setting it. The default is set where we need it further down the chain.

Reply:
Since we only call it from one place and the value is set. I will change the order and make it non optional. That should also solve your comment.